### PR TITLE
Adds actions - attempt 2

### PR DESCRIPTION
**Pull request by Crossedfall**
## About The Pull Request

We tried this before and it didn't really work. But, I'd like to give it another go. 

## Why It's Good For The Game

~~Automations are stealing our jerbs~~ Should help keep our repository a bit cleaner and more organized.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
```
Comment (qwertyquerty) [inline .github/workflows/stale.yml:8]: What is this builtin to?

Comment (Crossedfall) [reply .github/workflows/stale.yml:8]: I don't follow, what do you mean?

**Comment by Crossedfall** [comment]
~~The git action failure can be safely discarded, it failed because it knows that the changes would be overwritten by merging.~~

Edit: Here's a sample of it working - https://github.com/Crossedfall/BeeStation-Hornet/runs/470660800?check_suite_focus=true

Edit 2: Decided to just lock it to Master, since that makes the most sense.

**Comment by ike709** [comment]
What could go wrong?

**Comment by Crossedfall** [comment]
Note for downstreams:

The compile changelog and the stale action is worth porting (if you didn't already have similar actions). But, the docker one won't work for you unless you setup Docker Hub repository and add the required secrets to your github repo.

**Comment by MCterra10** [comment]
thanks for the info ghost boy

**Pull request merged**